### ISSUE TITLE
Possible improvement of an example in Basics > Pipe Operator

### ISSUE THREAD

**Issue by odiumediae** [doomspork/elixir-school#305]
One of the examples in the Pipe Operator section could be replaced, since:

```
"Elixir rocks" |> String.split |> Enum.map( &String.upcase/1 )
```

could simply be written:

```
iex> "Elixir rocks" |> String.upcase |> String.split
["ELIXIR", "ROCKS"]
```

which is not only shorter and more idiomatic, but also more readable. Under "Best Practices" the same example is used again, without the parentheses. A slightly different example might be helpful here. Maybe something like this:

```
# bad example
iex> "Elixir rocks" |> String.downcase |> String.split |> Enum.map &String.to_atom/1

#good example
iex> "Elixir rocks" |> String.downcase |> String.split |> Enum.map(&String.to_atom/1)

[:elixir, :rocks]
```

**Comment by doomspork** [comment]
@odiumediae please open a PR with your suggested changes 👍

**Comment by odiumediae** [comment]
[@doomspork](https://github.com/doomspork): I'd like to. Sorry, if this is a dumb question, but should I change the respective templates of all supported languages or just in the main (English) one?

Apropos languages, if I find the time, I'd like to provide the German translation. Any other German speakers here who'd like to help me?

**Comment by doomspork** [comment]
@odiumediae if you wouldn't mind making the example changes across all languages everyone would greatly appreciate that.

We have an issue [here](https://github.com/doomspork/elixir-school/issues/200) for German Translations, a few people have expressed interest but no one has started.

**Comment by doomspork** [comment]
This was addressed in #511  👍
